modules/apps/foundation/frontend-theme/space-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,216 @@
+                        font-size: 2.6rem;
+                        text-align: center;
+                        text-transform: uppercase;
+                        &:hover {
+                            color: $light-blue;
+                            text-decoration: none;
+                        }
+                    }
+                }  
+            }
+
+            @media screen and (max-width: 900px) {
+                h1 {
+                    display: block;
+                }
+
+                ul {
+                    list-style-type: none;
+                    margin: 0;
+                    padding: 0;
+                    width: 200px;
+
+                  li a {
+                      display: block;
+                      padding: 8px 16px;
+                      text-decoration: none;
+                  }
+                }
+            }
+        }
+
+    #content {
+        background-color: $background;
+        border: $dark-blue;
+        border-width: 3px;
+        color: $white;
+
+        #main-content {
+            border: $dark-blue;
+            border-width: 3px;
+
+            .portlet {
+                background-color: $light-blue;
+                border: $dark-blue;
+                border-width: 3px;
+            }
+        } 
+    }
+    
+    #footer {
+        background-color: #1F2021;
+        color: $light-blue;
+        height: 70px;
+        font-size: $s-size;
+        font-style: inherit;
+        font-weight: inherit;
+
+        p {
+            font-size: 16px;
+            padding-left: 80px;
+            padding-top: 15px;   
+        }
+    }   
+/*------------------------------------------------------------------NAV STYLING----------------------------------------------------------------*/
+    .portlet-content.portlet-content-editable {
+        background-color:$grey;
+        color: $off-black;
+
+        h2 {
+            text-transform: uppercase;
+        }
+    }
+
+    .portlet-content {
+        background-color: $grey;
+        color: $off-black;
+
+        h2 {
+            text-transform: uppercase;
+            &:hover {
+                color: $light-blue;
+                cursor: pointer;
+            }
+        }
+    }
+}
+
+/*--------------------------------------------------------------------Modal Styling------------------------------------------------------------------- */
+.modal {
+    background-color: rgba(0, 0, 0, 0.5);
+    display: none;
+    height: 100%;
+    left: 0; 
+    overflow: auto; 
+    position: fixed;
+    top: 0; 
+    width: 100%;
+    z-index: 1; 
+}
+
+.modal-content {
+    background-color: #FEFEFE;
+    border: 1px solid #888;
+    margin: 15% auto; 
+    padding: 20px;
+    width: 80%; 
+}
+
+.close {
+    color: #aaa;
+    float: right;
+    font-size: 28px;
+    font-weight: bold;
+    &:hover, &:focus {
+        color: black;
+        cursor: pointer;
+        text-decoration: none;
+    }
```

Comment:
Can't we nest these selectors in the above 3 CSS blocks together?  I assume the ```modal-content``` and ```close``` CSS class is resides within the ```modal``` CSS class?